servers/src/grin/seed.rs

```diff
@@ -300,11 +303,14 @@ fn listen_for_addrs(
 		return;
```

Comment:
I think we can remove the `peers.peer_count() > peers.peer_outbound_count()` check on line 302, since `healthy_peers_mix()` covers it.

Reply:
That makes sense. They don't quite check the same thing but I think we only want the `healthy_peers_mix()` check here as you said. I'm pretty sure we don't care to check here that the total connected peers exceeds the outbound connected peers. This would require (I think) that we have inbound connections, and I'm pretty sure we want to `return` here any time our outbound connections exceeds our configured preferred minimum outbound connections ("a healthy number of outbound peers").

Removed in https://github.com/mimblewimble/grin/pull/2985/commits/fb9b06ecf6f86a8424aa92f7a991b8754a8a0302 unless there is a good reason to keep this as is.